pkgs/by-name/by/byedpi/package.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "byedpi";
+  version = "0.13.1";
+
+  src = fetchFromGitHub {
+    owner = "hufrea";
+    repo = "byedpi";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-Sf5Ik8+9nKhum4/faGf44Y/gQggldyRsFUVmd9XPKOA=";
+  };
+
+  installPhase = ''
+    install -Dm755 ciadpi $out/bin/ciadpi
+  '';
+
+  strictDeps = true;
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "SOCKS proxy server implementing some DPI bypass methods";
+    homepage = "https://github.com/hufrea/byedpi";
+    changelog = "https://github.com/hufrea/byedpi/releases/tag/v${finalAttrs.version}";
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [ averyanalex ];
+    platforms = lib.platforms.all;
```

```suggestion
    platforms = with lib.platforms; linux ++ windows;
```

https://github.com/hufrea/byedpi/issues/17

The macros also only mentioned linux and windows.

---

Yes, looks like it only supports linux and windows, thanks